### PR TITLE
udiskslinuxlogicalvolume.c: Dupe code removal

### PR DESCRIPTION
**Pull request by tasleson**
Place common initialization and checking code into a function called `common_setup`.  I tried making a common function with numerous callbacks to virtually replace the implementation of all these functions with one, but that got too unwieldy so going with this simple change instead.

### CODE REVIEW

**Comment by vpodzime** [comment]
Jenkins, this is ok to test.

File: modules/lvm2/udiskslinuxlogicalvolume.c [hunk 1]
```diff
@@ -380,12 +376,42 @@ handle_delete (UDisksLogicalVolume   *_volume,
...
+  g_variant_lookup (options, "tear-down", "b", &teardown_flag);
+
+  if (!common_setup (volume,
+                     invocation,
+                     options,
+                     N_("Authentication is required to delete a logical volume"),
+                     &object,
+                     &daemon,
+                     &caller_uid,
+                     NULL))
```
Comment (vpodzime) [inline modules/lvm2/udiskslinuxlogicalvolume.c]: I think the first three arguments could go on a single line and so could the ``out_`` ones. But YMMV, as always. :)

Comment (tasleson) [inline modules/lvm2/udiskslinuxlogicalvolume.c]: @vpodzime Will update.  I was trying to match current code, but my preference would be to do as you suggest too.

**Comment by tasleson** [comment]
@vpodzime Updated the formatting as suggested, thanks for the review!

**Pull request merged**